packages/runner/__snapshots__/runner.mochaEvents.spec.js

```diff
@@ -14,7 +14,8 @@ exports['src/cypress/runner tests finish with correct state hook failures fail i
       "title": "",
```

Comment:
move this to `standard.mochaEvents.spec.js`

Reply:
punting on this to prevent significant rename + changes to the file